login.css

```diff
@@ -0,0 +1,24 @@
+.login {
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.login__box {
+    width: 20rem;
+    height: fit-content;
+    background-color: rgb(250, 245, 245);
```

Comment:
try to use variables for colors(so that you can use them else where easily)

Reply:
yes will try to use golbal variables